src/ui/public/field_editor/field_editor.js

```diff
@@ -37,7 +39,7 @@ uiModules
 
       self.docLinks = docLinks;
       getScriptingLangs().then((langs) => {
-        self.scriptingLangs = langs;
+        self.scriptingLangs = _.intersection(langs, ['expression', 'painless']);
```

Comment:
Can we DRY this up as well by re-using `getSupportedScriptingLangs()`? Maybe even factor out a `filterSupportedSciptingLangs(languages)` to encapsulate the `_.intersection` that is also used below? Or does that go too far?